src/mobile/src/components/Banner/BannerText.jsx

```diff
@@ -25,20 +25,41 @@ const styles = StyleSheet.create({
   },
 });
 
+async function getStudentQuotes() {
+  const { data, error } = await supabase.from('quotes').select('*');
+
+  if (error) {
+    throw new Error(error.message);
+  }
+
+  return data;
+}
+
 const BannerText = () => {
-  const [studentQuote, setStudentQuote] = useState(quotes[0]);
+  const [studentQuotes, setStudentQuotes] = useState({});
 
   useEffect(() => {
-    setStudentQuote(quotes[Math.floor(Math.random() * quotes.length)]);
+    getStudentQuotes()
+      .then((quotes) => {
+        const randomIndex = Math.floor(Math.random() * quotes.length);
+        setStudentQuotes(quotes[randomIndex]);
+        return quotes;
+      })
+      .catch((err) => console.error(err));
   }, []);
 
   return (
     <View style={styles.container}>
       <Text style={styles.logo}>Telescope</Text>
-      <Pressable style={styles.studentQuote} onPress={() => Linking.openURL(studentQuote.url)}>
-        <Text style={styles.studentQuoteText}>"{studentQuote.quote}"</Text>
-        <Text style={styles.studentQuoteText}> {studentQuote.author}</Text>
-      </Pressable>
+      {studentQuotes.length ? (
```

Comment:
Don't think you can do that on an object type.
```suggestion
      {studentQuotes ? (
```